src/rules/noUnboundMethodRule.ts

```diff
@@ -62,6 +66,7 @@ export class Rule extends Lint.Rules.TypedRule {
             The object may have three properties:
```

Comment:
Since it's no longer three, and keeping the number up-to-date is a bit of a pain...

```suggestion
             The object may have the following properties:
```